lara-typescript/src/interactive-api-client/api.ts

```diff
@@ -62,7 +64,11 @@ export const getInteractiveState = <InteractiveState>(): InteractiveState | null
 export const setInteractiveState = <InteractiveState>(newInteractiveState: InteractiveState | null) => {
   const client = getClient();
   client.managedState.interactiveState = newInteractiveState;
-  client.post("interactiveState", newInteractiveState);
+  window.clearTimeout(setInteractiveStateTimeoutId);
```

Comment:
If the interactive is constantly changing its state then the state will never be saved. I guess that is OK, because I can't of a case where that would be a normal.

Reply:
Right. There are two things that can help in this case:
- LARA has auto-save feature, it sends "getInteractiveState" every 2 or 5s (I don't remember now)
- it sends "getInteractiveState" before user leaves the page
So, the state still should be saved sooner or later.